src/main/java/com/google/javascript/clutz/Options.java

```diff
@@ -43,6 +43,10 @@
       usage = "run faster by skipping the externs parsing (useful for tests)")
   boolean skipParseExterns;
 
+  @Option(name = "--emitPlatformExterns",
+      usage = "emits platform externs.")
```

Comment:
clarify a bit more "emits platform externs, instead of omitting them in favor of TS lib.d.ts".